Build Infrastructure/README.md

```diff
@@ -0,0 +1,8 @@
+In order to build Infrastructure dlls you need to 
+1) checkout https://github.com/nasser/nostrand in this directory (read instructions in nostrand readme, (change target framework to net472 TODO, maybe optional check))
```

Comment:
Would better to have script and nostrand as submodule

Reply:
Would it be possible to just include the dlls artifacts?

Reply:
Also confused how I'm meant to target net 4.72 (msbuild trying to target 5.0 for me)

Reply:
Hi @selfsame ! 
Dlls already included, this instructions just in case if somebody will want to recompile

Currently nostrand target 4.71 and 5.0
https://github.com/nasser/nostrand/blob/bc359dc57449446a02dd52e0daa6758898557379/Nostrand.csproj#L5

so then I compiled nostrand dlls I changed it to 4.72 because default c# solution csproj generated by godot target net472, I'm actually have no idea is was it necessary or not, lol.
I will check when will find free time.

I think it is better convert this instruction in bash script and  add nostrand as submodule.